balor/balor.ino

```diff
@@ -83,14 +116,9 @@ void setup()
   }
   pinMode(LED_PIN, OUTPUT);
   digitalWrite(LED_PIN, HIGH);
-  int tmp = 0;
-  if (ros_nh.getParam("tmp_param", &tmp))
+  if (!getRequiredParams())
```

Comment:
What happens if we say `params_retrieved_successfully = getRequiredParams()` ?
If this doesn't work, we should document that in a comment here

Reply:
It does work. Just verified, and also made the change